docs/contributing/testing-guidelines/index.md

```diff
@@ -1,5 +1,23 @@
 # Testing guidelines
 
-!!! warning
+## Unit tests
 
-    Under Construction
+Unit Testing is a software testing method by which individual units of source code are tested to determine whether they are fit for use.
```

Comment:
- Follow https://developers.google.com/style/voice.
- I feel `fit for use` is uncler.

```suggestion
Unit testing is a software testing method that tests individual units of source code to determine whether they satisfy the specification.
```